Assignment Day 2 _9th December 2020.ipynb

```diff
@@ -24,7 +24,7 @@
     {
      "data": {
       "text/plain": [
-       "'Mary had a little lamb'"
+       "'Mary had a little lamb and also a girl with her'"
```

Comment:
**Suggestion**: Ensure that the updated string content does not impact any downstream analysis or output expectations within the notebook. If the string's content is used in examples, demonstrations, or tests further in the notebook, verify that these sections are reviewed and updated as necessary to reflect the change. <br> **Code Suggestion**: 
 ```
 No specific code change suggested, but a review of dependent cells is recommended.
 ```